plugins/node/opentelemetry-instrumentation-aws-sdk/test/dynamodb.test.ts

```diff
@@ -109,9 +115,7 @@ describe('DynamoDB', () => {
           expect(
             attrs[SemanticAttributes.AWS_DYNAMODB_PROJECTION]
           ).toStrictEqual('id');
-          expect(
-            JSON.parse(attrs[SemanticAttributes.DB_STATEMENT] as string)
-          ).toEqual(params);
+          expect(attrs).not.toHaveProperty(SemanticAttributes.DB_STATEMENT);
```

Comment:
Something missed, thanks for picking it up, will sort that out.

Reply:
In fact, since this is testing default behaviour and the upcoming change to not set the property if the serializer is not configured this can probably stay as is?